src/wp-includes/rest-api/endpoints/class-wp-rest-revisions-controller.php

```diff
@@ -254,7 +254,6 @@ public function get_items( $request ) {
 				'posts_per_page'   => -1,
 				'orderby'          => 'date ID',
 				'order'            => 'DESC',
-				'suppress_filters' => true,
```

Comment:
Why is this being removed?

Reply:
oops, thanks for spotting! was testing something unrelated